build.ps1

```diff
@@ -348,6 +348,10 @@ if ($RunBuild -and ($All -or $BuildJava) -and -not $NoBuildJava) {
         }
     }
 
+    if ($env:PATH -notlike "*${env:JAVA_HOME}*") {
```

Comment:
Clarify/comment on what this is for?

Reply:
It's a minor cleanup that fixes https://github.com/aspnet/AspNetCore-Internal/issues/2592.